docs/index.rst

```diff
@@ -6,31 +6,35 @@ Getting Started With CrateDB Cloud
 
 Welcome to CrateDB Cloud!
 
-CrateDB Cloud is a scalable SQL Platform-as-a-Service that allows for data
-ingestion, enrichment, analytics, visualization, and monitoring of massive
-amounts of machine and internet of things (IoT) data.
+The CrateDB Cloud platform is a scalable SQL cloud service that allows for
+real-time analytics and visualization of industrial internet of things (IIoT)
+data at massive scale. The cloud service is fully-managed and operated by our
+experts at Crate.io.
 
-Hosted and operated 24/7 by our experts, the fully-managed SQL cloud service is
-highly available, supports any type of data and BLOBs, provides
-full-text search, as well as aggregations and Joins.
+This step-by-step tutorial shows you how to:
 
-This step-by-step tutorial helps you to get started with CrateDB Cloud. You
-will learn how to:
-
-- Create your CrateDB Cloud user account and install the Croud CLI
-- Use Croud to create your first organization, project, and resources
+- Create a CrateDB Cloud user account
+- Install the Croud CLI, a command-line interface tool that lets you interact
+  with the system
 - Deploy your first CrateDB cluster
 
 .. NOTE::
 
    This is beginner documentation.
```

Comment:
this section should link through to the complete reference manuals (Croud, the console) for CrateDB Cloud, like the CrateDB getting started docs index page does for the CrateDB reference and guide